src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import './Loader.scss';
+
+export const Loader = () => (
+  <div className="Loader" data-cy="loader">
+    <div className="Loader__content" />
```

Comment:
```HTML
<div className="Loader__content" />
```
There are no reason to using BEM because you had chosen css modules.

Reply:
I like to use BEM because it simplify code and facilitate refactoring.